<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>651</width>
     <height>521</height>
    </rect>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>251</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>631</width>
       <height>251</height>
      </rect>
     </property>
     <property name="title">
      <string>Consulta</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>40</y>
        <width>151</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLineEdit" name="texto_numregistro"/>
       </item>
       <item>
        <widget class="QLineEdit" name="texto_clave"/>
       </item>
       <item>
        <widget class="QLineEdit" name="texto_volumen"/>
       </item>
       <item>
        <widget class="QComboBox" name="cbo_editorial"/>
       </item>
       <item>
        <widget class="QLineEdit" name="texto_procedencia"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>68</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Id:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Titulo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Autor:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>AÃ±o:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>40</y>
        <width>84</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>No. registro</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Clave</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Volumen</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Editorial</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Procedencia</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>161</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="texto_id"/>
       </item>
       <item>
        <widget class="QLineEdit" name="texto_titulo"/>
       </item>
       <item>
        <widget class="QLineEdit" name="texto_autor"/>
       </item>
       <item>
        <widget class="QLineEdit" name="texto_edicion_anio"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="boton_buscar">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>90</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="boton_cerrar">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>140</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Cerrar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="texto_dia">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>220</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="texto_mes">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>220</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="texto_anio_">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>220</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>0</string>
      </property>
     </widget>
     <zorder>layoutWidget_2</zorder>
     <zorder>layoutWidget_3</zorder>
     <zorder>layoutWidget_4</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>boton_buscar</zorder>
     <zorder>boton_cerrar</zorder>
     <zorder>texto_dia</zorder>
     <zorder>texto_mes</zorder>
     <zorder>texto_anio_</zorder>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>631</width>
      <height>251</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableView" name="tabla_datos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>611</width>
       <height>221</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
